### DISCUSSION TITLE
第6节课笔记（23班）

### DISCUSSION THREAD

**Discussion by wangyijunlyy** [InternLM/Tutorial#420] [笔记]
笔记提交

**Comment by PlateAss** [comment]
https://plateass.github.io/class6

**Comment by Epi10gue** [comment]
http://t.csdnimg.cn/tFVER

**Comment by i-square** [comment]
https://i-square.github.io/p/InternLM-tutorial-campsection6-OpenCompass-and-LLM-Evaluation/
